### PR TITLE
Adding kafka verticle for interacting with kafka message queue

### PR DESCRIPTION
**Pull request by ShamsUlAzeem**
Setup with environment and system property variables for initiating a connection with a kafka message queue

### CODE REVIEW

File: konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/settings/KonduitSettings.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
+package ai.konduit.serving.pipeline.settings;
```
Comment (agibsonccc) [inline konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/settings/KonduitSettings.java:1]: Copyright

File: konduit-serving-vertx-protocols/konduit-serving-kafka/src/main/java/ai/konduit/serving/vertx/protocols/kafka/VertxKafkaModuleInfo.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package ai.konduit.serving.vertx.protocols.kafka;
+
+import ai.konduit.serving.annotation.module.ModuleInfo;
+
```
Comment (agibsonccc) [inline konduit-serving-vertx-protocols/konduit-serving-kafka/src/main/java/ai/konduit/serving/vertx/protocols/kafka/VertxKafkaModuleInfo.java:4]: Copyright

File: konduit-serving-vertx-protocols/konduit-serving-kafka/src/main/java/ai/konduit/serving/vertx/protocols/kafka/verticle/InferenceVerticleKafka.java [hunk 3]
```diff
@@ -0,0 +1,240 @@
+package ai.konduit.serving.vertx.protocols.kafka.verticle;
+
```
Comment (agibsonccc) [inline konduit-serving-vertx-protocols/konduit-serving-kafka/src/main/java/ai/konduit/serving/vertx/protocols/kafka/verticle/InferenceVerticleKafka.java:2]: Copyright

**Review by agibsonccc — CHANGES_REQUESTED** [review]
Minor nits (copyright)

**Comment by ShamsUlAzeem** [comment]
@agibsonccc sorry about that new not sure why the IDE didn't auto add them

**Review by agibsonccc — APPROVED** [review]

**Pull request merged**